yenly/smart-shopping-list
# Feat connect to firebase

## Description
- Installed `firebase` and `react-firebase-hooks` packages and set up this app to connect to firebase.
- Created example message component to add data to and read data from firebase.

## Related Issue

Closes #1 

## Acceptance Criteria


## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |




## Testing Steps / QA Criteria

1. run app locally
2. enter  your message in input field, click send button
3. check to see message appear on list below.
4. log in to https://console.firebase.google.com,
5. under messages collection, create a new document with your new message
6. check your app running on localhost to see new message
